src/libs/algolia/fixtures/algoliaFixtures.ts

```diff
@@ -95,6 +100,7 @@ export const mockedAlgoliaResponse = toMutable({
         postalCode: '75000',
         city: 'Paris',
       },
+      artists: [{ id: '4', name: 'Artist 4' }],
```

Comment:
```suggestion
      artists: [{ id: '4', name: 'Artist 4' }, { id: '5', name: 'Artist 4' }],
```
Pour tester le `uniqBy`